WalletWasabi.Fluent/ViewModels/Wallets/Send/LabelSelectionViewModel.cs

```diff
@@ -111,6 +111,8 @@ private bool TryGetBestKnownByRecipientPockets(Pocket[] knownByRecipientPockets,
 				.ToArray();
 
 		var bestPockets = new List<Pocket>();
+		bestPockets.AddRange(privateAndSemiPrivatePockets);
```

Comment:
Since we will use `Except` later why not add `privateAndSemiPrivatePockets` at the end (L138)?

Reply:
Its amount has to be taken into account during calculations at L120 and L128.

`Except` is used because the private and semi private coins shouldn't be removed in the case of over selection.

Reply:
Ok, thanks.
It would be nice to add a comment on top of each foreach loop to explain what it does.

Reply:
Done